### PR TITLE
add Caution note about deprecation

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- add Caution note about deprecation
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: docs/03-03-examples-alpha.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ title: Kyma CLI alpha command usage examples
 type: Details
 ---
 
+> **CAUTION:** The following information is deprecated. To download the stable binaries, follow the instructions in the [CLI README file](https://github.com/kyma-project/cli#readme).
```
Comment (NHingerl) [inline docs/03-03-examples-alpha.md:6]:
```suggestion
> **CAUTION:** The following information is outdated. If you'd like to try out the commands, install the stable Kyma CLI from the Kyma main branch. To download the stable binaries, follow the instructions in the [CLI README file](https://github.com/kyma-project/cli#readme).
```

**Review by a-thaler — DISMISSED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Comment by NHingerl** [comment]
@adamwalach helps fixing the error that blocks the merge

**Comment by NHingerl** [comment]
replaced by https://github.com/kyma-project/cli/pull/990

**Pull request closed without merging**